### ISSUE TITLE
Infobox transclusion

### ISSUE THREAD

**Issue by alanguir** [Credera/angular-bing-maps#24]
I was hoping to be able to transclude arbitrary content into the infobox. I'm hacking on it now, but can't really figure out where the markup for the infobox is coming from. Does bing render that from JS? Have this been tried?

**Comment by sjmorrow** [comment]
You are correct, Bing DOES render that from the JS. I spent a few hours trying to get this to work initially, but I'm afraid there is no direct way to get an angular linked template into a Bing Maps Infobox. Adding HTML content to an Infobox IS fairly straight forward, however that HTML will lose all data binding and other angular functionality.

Your PR (#25) is a great starting point.

The fully functional solution to this problem involves the user passing in a scope AND an HTML template, watching for changes on that scope, recompiling and linking the HTML template, and then inserting that newly compiled element's HTML into the Bing InfoBox. All-in-all, a completely feasible solution that requires a fair amount of development time.
